services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -25,130 +27,143 @@ export default class IndexPage extends React.Component<Props, State> {
+
         <div className="b b-c">
           <SectionHeader title="Commissions Page Scaffold" />
-          <form className="m-t700" action="javascript:void(0)" method="POST">
-            <fieldset className="fs m-v700">
-              <legend className="fs-l">Application Information</legend>
-              <div className="fs">
-                <div className="fs-c">
-                  <TextInput
-                    title="First Name"
-                    name="firstName"
-                    placeholder="First Name"
-                    required
-                  />
-                  <TextInput
-                    title="Middle Name"
-                    name="middleName"
-                    placeholder="Middle Name"
-                    required
-                  />
-                  <TextInput
-                    title="Last Name"
-                    name="lastName"
-                    placeholder="Last Name"
-                    required
-                  />
-                  <TextInput
-                    title="Address"
-                    name="address"
-                    placeholder="Address"
-                    required
-                  />
-
-                  <TextInput
-                    title="Unit"
-                    name="unit"
-                    placeholder="Unit or Apartment # "
-                    required
-                  />
-
-                  <TextInput
-                    title="City"
-                    name="city"
-                    placeholder="City"
-                    required
-                  />
-
-                  <TextInput
-                    title="State"
-                    name="state"
-                    placeholder="State"
-                    required
-                  />
-
-                  <TextInput
-                    title="Zip"
-                    name="zip"
-                    placeholder="Zip Code"
-                    required
-                  />
-
-                  <TextInput
-                    title="Phone"
-                    name="phone"
-                    placeholder="Phone Number"
-                    required
-                  />
 
-                  <TextInput
-                    title="Email"
-                    name="email"
-                    placeholder="Email "
-                    required
-                  />
+          <Formik
+            initialValues={{
+              firstName: '',
+              middleName: '',
+              lastName: '',
+              address: '',
+              unit: '',
+              state: '',
+              city: '',
+              zip: '',
+              phone: '',
+              email: '',
+              confirmEmail: '',
+            }}
+            validationSchema={Yup.object().shape({
+              zip: Yup.string()
+                .required('Zip Code Is Required')
+                .min(5, 'Zip Codes Contains 5 Digits'),
+            })}
+            onSubmit={values => console.log(values)}
```

Comment:
just change this to `onSubmit={() => {}}` so you don't get linting errors due to having a `console.log`